account/handler.go

```diff
@@ -0,0 +1,58 @@
+package account
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/ayushjnv1/Gobank/api"
+	"github.com/gorilla/mux"
+)
+
+func CreateAccount(service Service) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		var account AccountCreate
+		err := json.NewDecoder(r.Body).Decode(&account)
+		if err != nil {
+			api.Error(w, http.StatusBadRequest, api.Response{Message: err.Error()})
+			return
+		}
+		cus, err := service.CreateAccount(r.Context(), account.UserID)
```

Comment:
define a cctx varriable at the start of the method, so that whereever we need context value we can directly use that variable